crates/cosmos/cosmos-integration-tests/src/traits/bootstrap/build_chain.rs

```diff
@@ -16,5 +17,6 @@ where
         chain_node_config: &Self::ChainNodeConfig,
         chain_genesis_config: &Self::ChainGenesisConfig,
         relayer_wallet: &WalletOf<Self::Chain>,
+        eip_query_type: EipQueryType,
```

Comment:
We don't need this extra argument in general. We just need it in `BuildCosmosChainWithNodeConfig`, which we can get by requiring `Bootstrap: HasEipQueryType`.

Reply:
Done in 734aa95eced3f6bef7030bc7e68888ecdca2179b